core/src/test/java/org/infinispan/configuration/EvictionConfigurationTest.java

```diff
@@ -49,7 +50,7 @@ public void testReuseLegacyBuilder() {
 
       assertEquals(configuration.memory().maxSizeBytes(), -1);
       assertEquals(configuration.memory().maxCount(), 200);
-      assertEquals(configuration.memory().storageType(), OBJECT);
+      assertEquals(configuration.memory().storageType(), HEAP);
```

Comment:
This is a breaking change, as in 10.0.0, there is no ```HEAP``` enum constant.

Reply:
Maybe we could re-evaluate keeping the old stuff around and clean everything up for sake of Maintainability